ci-operator/config/openshift/release/openshift-release-master__nightly-4.20.yaml

```diff
@@ -1252,18 +1252,6 @@ tests:
       enable:
       - observers-resource-watch
     workflow: openshift-e2e-gcp-etcd-scaling
```

Comment:
you should probably remove it from [4.21](https://github.com/openshift/release/blob/e7a6f5ec792fb2a39497a62bd0ab6a2a05520762/ci-operator/config/openshift/release/openshift-release-master__nightly-4.21.yaml#L1255) as well